diff --git a/usr/src/supervisor/supervisor/__main__.py b/usr/src/supervisor/supervisor/__main__.py
index 686e83f09..00b2832f7 100644
--- a/usr/src/supervisor/supervisor/__main__.py
+++ b/usr/src/supervisor/supervisor/__main__.py
@@ -38,6 +38,8 @@ def run_os_startup_check_cleanup() -> None:
 if __name__ == "__main__":
     bootstrap.initialize_logging()
 
+    _LOGGER.warning("Patched supervisor. for reference: https://github.com/qweritos/haos-one")
+
     # Init async event loop
     loop = asyncio.new_event_loop()
     asyncio.set_event_loop(loop)
